app/controllers/answers_controller.rb

```diff
@@ -30,19 +30,46 @@ def create
   end
 
   def edit
-    check_your_privilege('Edit', @answer)
   end
 
   def update
-    return unless check_your_privilege('Edit', @answer)
+    return if current_user.nil?
```

Comment:
This line is unnecessary - this action is behind a `before_action :authenticate_user!`, so it's not possible to access this without being signed in.